modules/veteran/app/models/veteran/user.rb

```diff
@@ -44,17 +45,9 @@ def previous_poa_code
       @previous_poa_code = poa_codes.delete_if { |poa_code| poa_code == current_poa_code }.first
     end
 
-    def bgs_service
+    def bgs_org_service
       external_key = "#{@user.first_name} #{@user.last_name}"
-      @bgs_service ||= BGS::Services.new(
-        external_uid: @user.mpi_icn,
-        external_key: external_key.presence || @user.mpi_icn
-      )
-    end
-
-    def local_bgs_service
-      external_key = "#{@user.first_name} #{@user.last_name}"
-      @local_bgs_service ||= ClaimsApi::LocalBGS.new(
+      @bgs_itf_service ||= ClaimsApi::OrgWebService.new(
```

Comment:
I always question why the choice was made to use different identifiers here - I know this isn't your change, but bgs-ext used the [values passed to the initializer](https://github.com/department-of-veterans-affairs/bgs-ext/blob/350e45ae699407948d254d6fadd4f1b249e996f5/lib/bgs/base.rb#L45-L46), which come from [here](https://github.com/department-of-veterans-affairs/vets-api/blob/e55832144304b527606946a6bdf6fdc3fb8583b3/config/initializers/bgs.rb#L15-L16)

Reply:
Discussed with Alex and we plan to revisit and potentially modify the approach used here and in other places in our code.